src/mumble/Log.cpp

```diff
@@ -598,6 +617,16 @@ void Log::log(MsgType mt, const QString &console, const QString &terse, bool own
 #endif
 }
 
+void Log::processDeferredLogs() {
+	QMutexLocker mLocker(&Log::qmDeferredLogs);
+
+	while (!qvDeferredLogs.isEmpty()) {
+		LogMessage msg = qvDeferredLogs.takeFirst();
+
+		this->log(msg.mt, msg.console, msg.terse, msg.ownMessage, msg.overrideTTS);
```

Comment:
We never use the `this` specifier, unless required.

Reply:
Can't say that I like that a lot, but I'll change it for now...